### PR TITLE
refactor: remove colors dependency

### PR DESCRIPTION
**Pull request by vchimev**
https://github.com/NativeScript/nativescript-dev-appium/issues/51

### CODE REVIEW

**Comment by SvetoslavTsenov** [comment]
Hey, @vchimev, can you try to fix the dependency. I think it is time to use it in our log for better visualization.
Here is an example of [clors](https://www.npmjs.com/package/colors)

` 
var colors = require('colors');
 
colors.setTheme({
  info: 'blue',
  data: 'grey',
  help: 'cyan',
  warn: 'yellow',
  error: 'red'
});
 
// outputs red text 
console.log("this is an error".error);
 
// outputs yellow text 
console.log("this is a warning".warn);`

Basically, we can use red for errors, orange for warning, blue for basic info and etc.

**Comment by vchimev** [comment]
The `colors` dependency has been removed in https://github.com/NativeScript/nativescript-dev-appium/pull/63.
